### ISSUE TITLE
Search Lessons is Broken

### ISSUE THREAD

**Issue by vkoves** [Tweechable/tweechable_final#34]
The search feature on the [browse lessons page](https://tweechable.herokuapp.com/lessons) is broken and returns the default Heroku 500 error page. See [this example](https://tweechable.herokuapp.com/lessons?utf8=%E2%9C%93&keyword=test). The search should probably be disabled until the issue can be resolved.

**Comment by vkoves** [comment]
I've fixed this issue in a branch I'm working on called [`lessons-search-fix`](https://github.com/Tweechable/tweechable_final/tree/lessons-search-fix), where I fixed the issue by applying pagination to the lessons when they are the result of search. More specifically, I added `.paginate` to `@lessons` in `lessons_controller.rb`. I would also like to add a message if the search returns no results, and indicate what you are searching for (as checking the URL to see what search query is being used isn't very effective). Let me know if you have any feedback on this or want to help out.
